### PR TITLE
feat: first draft for logger

### LINKED ISSUE #445: Pick and use a logging library
### Describe the chore

Pick and use a logging library. The decision of what specific library or self written solution we use, should be made in this **issue**.

Suggestions:
- https://pub.dev/packages/logger

### PR DESCRIPTION
**Pull request by florian-sabonchi**
## Which issues does this pull request close?
closes #445

### CODE REVIEW

File: apps/tasks/lib/controllers/assignee_select_controller.dart [hunk 1]
```diff
@@ -41,13 +42,15 @@ class AssigneeSelectController extends ChangeNotifier {
...
+  final Logger _logger = Logger();
+
   /// Loads the tasks
   Future<void> load() async {
     state = LoadingState.loading;
     String? currentOrganization = CurrentWardService().currentWard?.organizationId;
     if(currentOrganization == null){
       if(kDebugMode){
-        print("Organization Id not set in CurrentWardService while trying to load in AssigneeSelectController");
+        _logger.w("Organization Id not set in CurrentWardService while trying to load in AssigneeSelectController");
```
Comment (DasProffi) [inline apps/tasks/lib/controllers/assignee_select_controller.dart:53]: Unsure whether below is a better alternative, here at least we don't have to declare a logger on every Object, or maybe we can use a logger service
```suggestion
        Logger().w("Organization Id not set in CurrentWardService while trying to load in AssigneeSelectController");
```

**Review by DasProffi — APPROVED** [review]

File: apps/tasks/lib/controllers/assignee_select_controller.dart (hunk 1 above)
Comment (florian-sabonchi) [reply apps/tasks/lib/controllers/assignee_select_controller.dart:53]: > Unsure whether below is a better alternative, here at least we don't have to declare a logger on every Object, or maybe we can use a logger service

I think a logger service would be the best

**Comment by MaxSchaefer** [comment]
@florian-sabonchi why was this pr merged with failed checks?

**Comment by florian-sabonchi** [comment]
> @florian-sabonchi why was this pr merged with failed checks? ... and how ... ?

it failed because i accepted suggestion of DasProffi and merged without waiting for the pipeline

**Pull request merged**